src/uneditable-section/section-item.styles.tsx

```diff
@@ -37,3 +41,63 @@ export const Container = styled.li<ContainerStyleProps>`
         grid-column: auto / span 4;
     }
 `;
+
+export const IconContainer = styled.div`
+    display: flex;
+    height: 100%;
+    align-items: center;
+    justify-content: center;
+    color: ${Color.Primary};
+    margin-left: 0.5rem;
+
+    svg {
+        width: 1.125rem;
+        height: 1.125rem;
+    }
+`;
+
+export const Clickable = styled.button`
```

Comment:
we can apply a text overflow wrap here to handle long text
![Screenshot 2024-03-06 at 9 23 01 AM](https://github.com/LifeSG/react-design-system/assets/16853062/5b940ef6-2394-4987-b199-31466da9876e)

for such scenarios, is the icon expected to remain on the right?

Reply:
When you say expected to remain on the right, are u referring to alongside the overflowed text?

Reply:
referring to columned layout vs inlined